### ISSUE TITLE
Request : dependency versions in release log

### ISSUE THREAD

**Issue by dorpsidioot** [payara/Payara#1039]
The current payara release logs have a listing of the updated dependencies, but it would be nice if we also had a comprehensive list of the current dependencies, makes it easier to update pom versions to support the newer libraries. If it already exists, excuse me, haven't found it yet.

**Comment by mulderbaba** [comment]
Payara has a comprehensive set of dependencies and they are being updated accordingly through its release cycle by applying internal QA procedures. I'd be not in favour of updating the whole dependency set to its latest version at once, which could potentially introduce problems. You can easily list the whole dependency graph by executing `mvn dependency:tree` or `mvn dependency:list` on the parent pom. You can also create a convergence report by executing `project-info-reports:dependency-convergence` that you'll find under target/site folder after execution.

My suggestion here would be for you is to list which versions of the artefacts you want to upgrade with the cause of your need and have the Payara team working on it.

**Comment by dorpsidioot** [comment]
Maybe I wasn't clear, but it was just a request for knowing the current versions of the dependencies in each version, not to force an update of all the existing dependencies. If you look at the release logs you put out, you always list the updated libraries/dependencies, but not the current state of all of them. So either I have to check the dependencies of the payara version from the git or traceback all the previous releases to check for possible changes.

**Comment by mulderbaba** [comment]
@dorpsidioot thanks for clarification, informing @mikecroft that our users requesting a page where we list the whole dependency graph of Payara with stating GAV coordinates. In the meantime @dorpsidioot, I hope aforementioned maven commands could be of any help to identify the versioning. If you have a specific question on an artefact's versioning throughout the release cycle of Payara, feel free to respond.

**Comment by mikecroft** [comment]
Yep, good suggestion. We'll have a look at where is best to put it.
I've created `PYSVC-82` to look at adding a page to the docs.

**Comment by dorpsidioot** [comment]
Thanks! Will make my life a lot easier ;-)
